netket/stats/mc_stats.py

```diff
@@ -80,6 +113,12 @@ def __repr__(self):
             ext = ", R̂={:.4f}".format(self.R_hat)
         else:
             ext = ""
+        if not config.FLAGS["NETKET_EXPERIMENTAL_FFT_AUTOCORRELATION"]:
+            if not math.isnan(self.tau_corr):
+                ext += ", τ={:.1f}".format(self.tau_corr)
+        elif not (math.isnan(self.tau_corr) and math.isnan(self.tau_corr_max)):
+            ext += ", τ={:.1f}<{:.1f}".format(self.tau_corr, self.tau_corr_max)
+
```

Comment:
I suggest we keep the old behavior when the flag is not set, which was not printing tau in the here at all.
```suggestion
        if config.FLAGS["NETKET_EXPERIMENTAL_FFT_AUTOCORRELATION"]:
            if not (math.isnan(self.tau_corr) and math.isnan(self.tau_corr_max)):
                ext += ", τ={:.1f}<{:.1f}".format(self.tau_corr, self.tau_corr_max)
```